### PR TITLE
KNDP-56: fixed wrong OCI image names

### PR DESCRIPTION
**Pull request by catalindev0ps**

### CODE REVIEW

File: .github/workflows/releases.yaml [hunk 1]
```diff
@@ -55,28 +57,49 @@ jobs:
...
+
+      - name: delete build.txt
+        run: rm build.txt
 
+      - name: Apply new changes to current branch
+        run: |
+            git remote set-url origin https://x-access-token:${{ secrets.GITHUB_TOKEN }}@github.com/${{ github.repository }}
+            git checkout ${GITHUB_REF_NAME}
+            git add .
+            git commit -m "changes from action"
```
Comment (art10m) [inline .github/workflows/releases.yaml:101]: @catalindev0ps Please change to "Updated appVersion by action"

File: packages/kndp/charts/kndp/values.yaml [hunk 2]
```diff
@@ -75,7 +75,7 @@ ingress-nginx:
       publish-status-address: localhost
       enable-ssl-passthrough: true
 
- 
+
```
Comment (art10m) [inline packages/kndp/charts/kndp/values.yaml:78]: @catalindev0ps Remove extra line

File: .github/workflows/releases.yaml [hunk 3]
```diff
@@ -55,28 +57,49 @@ jobs:
...
+      - name: Updating charts appVersion
+        run: cat build.txt | xargs -I{} bash -c '[ -d {}/charts ] && yq -i ".appVersion=\"${{ github.sha }}\"" {}/charts/*/Chart.yaml || true'
+
+
       - name: Build images
         id: build
 
         run: cat build.txt | xargs -I{} bash -c '[ -f {}/Dockerfile ] && docker build -t ${REGISTRY}/${{ github.repository }}/$(basename {}):${{ github.sha }} -f {}/Dockerfile . && docker push ${REGISTRY}/${{ github.repository }}/$(basename {}):${{ github.sha }} || true'
 
+
```
Comment (art10m) [inline .github/workflows/releases.yaml:83]: @catalindev0ps Remove extra line

File: .github/workflows/releases.yaml [hunk 4]
```diff
@@ -55,28 +57,49 @@ jobs:
...
+      - name: install yq
+        run: | 
+            sudo wget -qO /usr/local/bin/yq https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64
+            sudo chmod a+x /usr/local/bin/yq
+
+      - name: Updating charts appVersion
+        run: cat build.txt | xargs -I{} bash -c '[ -d {}/charts ] && yq -i ".appVersion=\"${{ github.sha }}\"" {}/charts/*/Chart.yaml || true'
+
+
```
Comment (art10m) [inline .github/workflows/releases.yaml:77]: @catalindev0ps Remove extra line

**Review by art10m — CHANGES_REQUESTED** [review]

**Comment by web7evgheni** [comment]
@catalindev0ps Please make explainable titles of PR, because nobody knows what you what to do by this PR.

**Comment by web7evgheni** [comment]
Now chart appVersion used for updating image tag dynamically is using correct tag for each built image.

**Review by web7evgheni — APPROVED** [review]

**Pull request merged**